src/audit/audit_test.go

```diff
@@ -49,9 +49,9 @@ func TestAudit(t *testing.T) {
 		threadID := uint32(i)
 		query := "select a,b,cd from table1 where a=b and c=d and e=d group by id order\n by desc"
 		if i%2 == 0 {
-			audit.LogWriteEvent(typ, user, host, threadID, query, 0, time.Now())
+			audit.LogWriteEvent(typ, user, host, threadID, query, 0, 0, time.Now())
```

Comment:
I think we use `false` instead of `0` is better to understand the meaning of the parameters here.

Reply:
> I think we use `false` instead of `0` is better to understand the meaning of the parameters here.

Currently status is only `0` or `1`, but in the future, if we use errno just like MySQL, there may be other values.

Reply:
> > I think we use `false` instead of `0` is better to understand the meaning of the parameters here.
> 
> Currently status is only `0` or `1`, but in the future, if we use errno just like MySQL, there may be other values.
> 
> Such as:
> In MySQL, the error info `[Err]1046-No database selected`, the `status` in `audit log` is `1046`.

Make sense.